samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java

```diff
@@ -1018,6 +1019,12 @@ private ContentsDeliveryBean getPageContentsByQuestion(
         } else {
           delivery.setPrevious(false);
         }
+        //linear access
+        if (delivery.getNavigation().equals("1")) {
```

Comment:
Invert it or use a variable

Reply:
Added a variable. This variable should be used in all other cases, but I don't want to modify unrelated code